### ISSUE TITLE
Create facet: Date range = dc:date (multi-valued)

### ISSUE THREAD

**Issue by marlo-longley** [sul-dlss/vt-arclight#141]
Ref #29 

Discussion:
We want to change this label to Year, is that OK?
Should we support BL Range Limit? -- this doesn't make tons of sense since we have a 2 year span. Range Limit only works on years.

**Comment by ggeisler** [comment]
Why are we restricted to only year for Date?

I haven't been following the details of the ASpace work but in @laurensorensen's original spreadsheet at least, the `dc:date` column includes dates in this format `1945-11-21`.

**Comment by jcoyne** [comment]
@ggeisler currently we're using the Arclight Indexer which was built for blacklight range limit.  Blacklight range limit only handles years, so the indexer only indexed years.  We're looking for more details about what the requirements are around this facet.

**Comment by ggeisler** [comment]
Okay, I see. 

If we are restricted by year, then a date facet of any kind is probably not super useful (I'm not sure how often a user will just want the 1945 documents and not the 1946 ones, but maybe I guess. But not with the BL Range Limit, I agree).

If the BL Range Limit facet could work with more specificity (at least months), that would likely be pretty useful for NTA users. Focusing on a certain period of a few months, for example, seems potentially something a researcher might want to do.

**Comment by jcoyne** [comment]
Currently it looks like we're only finding dates in 8 of 461 items
<img width="328" alt="Screenshot 2022-11-02 at 3 12 58 PM" src="https://user-images.githubusercontent.com/92044/199593378-769937c5-c3b8-4b7c-a34b-4bd074b99af2.png">

**Comment by marlo-longley** [comment]
@laurensorensen is this expected ^^ re only having dates for 8 items?

**Comment by marlo-longley** [comment]
Will we have dates for each item in the collection? No.
Would date facets even be useful in that case?
**TODO: investigate what date data we have available. This will influence UI design.**

What is the granularity of date faceting that we want, Year, Month ...?
We aren't going to use BL Range Limit because it only handles Years.
We could still somehow do more granular filtering for this project using a custom approach.

What does a good UI look like for the dates that we will have?

**Comment by marlo-longley** [comment]
Blocked by #157

**Comment by marlo-longley** [comment]
Closing in favor of hierarchical date facet strategy.
